src/components/CompareResults/Retrigger/RetriggerButton.tsx

```diff
@@ -156,14 +178,28 @@ function RetriggerButton({ result }: RetriggerButtonProps) {
 
   return (
     <>
-      <IconButton
-        title={Strings.components.revisionRow.title.retriggerJobs}
-        color='primary'
-        size='small'
-        onClick={() => void onRetriggerButtonClick()}
-      >
-        <RefreshOutlinedIcon />
-      </IconButton>
+      {variant === 'text' ? (
+        <Button
+          title={Strings.components.revisionRow.title.retriggerJobs}
+          className={styles.retrigger}
```

Comment:
I'd remove all styles, they don't seem useful;

Only the width is problematic, but I have another suggestion (see the main comment)